packages/@aws-cdk/aws-eks/lib/cluster.ts

```diff
@@ -609,6 +633,8 @@ abstract class ClusterBase extends Resource implements ICluster {
   public abstract readonly kubectlEnvironment?: { [key: string]: string };
   public abstract readonly kubectlSecurityGroup?: ec2.ISecurityGroup;
   public abstract readonly kubectlPrivateSubnets?: ec2.ISubnet[];
+  public abstract readonly clusterOpenIdConnectIssuerUrl: string;
+  protected openIdConnectProviderInstance?: iam.IOpenIdConnectProvider;
```

Comment:
These are not needed here anymore. The imported cluster will always accept a provider in the attributes, so for that case, the provider is just like a regular property.